### PR TITLE
Feature/play pause video creator

### PR DESCRIPTION
**Pull request by Kartm**

### CODE REVIEW

File: app/src/main/res/layout/fragment_creator.xml [hunk 1]
```diff
@@ -125,17 +125,17 @@
...
+            android:id="@+id/saveButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:background="?android:selectableItemBackground"
+            android:drawableLeft="@drawable/icon_save"
+            android:text="@string/save_button_text"
+            android:textColor="@color/secondaryColor"
+            android:textSize="16sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
```
Comment (Kaspr-whtev) [inline app/src/main/res/layout/fragment_creator.xml]: Why did you uncomment this?

**Review by Kaspr-whtev — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_creator.xml (hunk 1 above)
Comment (Kartm) [reply app/src/main/res/layout/fragment_creator.xml]: I needed a button to save a video, and there was none so I decided to uncomment and use this one

Comment (Kaspr-whtev) [reply app/src/main/res/layout/fragment_creator.xml]: You don't have this button?

image

Comment (Kartm) [reply app/src/main/res/layout/fragment_creator.xml]: No I don't. I must've accidentally removed it on my branch somehow, I'll check

Comment (Kaspr-whtev) [reply app/src/main/res/layout/fragment_creator.xml]: from what I can see, you just deleted all code regarding the action bar button

Comment (Kartm) [reply app/src/main/res/layout/fragment_creator.xml]: Apparently I made a mess when updating my branch to main.

It's fixed now!

image

**Review by Kaspr-whtev — APPROVED** [review]

**Pull request merged**